xml/repeated-content-decl.ent

```diff
@@ -2015,3 +2015,12 @@ User:           username@ldap.example.com
    information see <xref linkend="cha-cap-depl-notes-releases"/>.
   </para>
  </warning>'>
+
+<!--ENTITY values-file-changes...............................................-->
+
+<!ENTITY values-file-changes
+'<para xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+  The format of the &values-file; file has been restructured completely.
```

Comment:
Need to update to `&values-filename` 

I updated the entity so `&values-file` is just the name of the file and `&values-filename` will have the filename tag. E.g. `<filename>kubecf-config-values.yaml</filename>`